src/kontrol/__main__.py

```diff
@@ -375,6 +376,16 @@ def exec_unrefute_node(foundry_root: Path, test: str, node: NodeIdLike, version:
     foundry_unrefute_node(foundry=_load_foundry(foundry_root), test=test, node=node, version=version)
 
 
+def exec_split_node(
+    foundry_root: Path, test: str, node: NodeIdLike, branch_condition: str, version: int | None, **kwargs: Any
+) -> None:
+    node_ids = foundry_split_node(
+        foundry=_load_foundry(foundry_root), test=test, node=node, branch_condition=branch_condition, version=version
+    )
+
+    print(f'Node {node} has been split into {node_ids} on condition {branch_condition}.')
```

Comment:
Seems like this message is duplicated. It's present here and in `foundry_split_node`. Is this intended?